sbncode/TPCReco/CalorimetryAnalysis/run_calorimetry_analysis_icarus_sce_multistream.fcl

```diff
@@ -0,0 +1,160 @@
+#include "services_common_icarus.fcl"
+
+#include "larproperties.fcl"
+#include "backtrackerservice.fcl"
+#include "particleinventoryservice.fcl"
+#include "detectorproperties_icarus.fcl"
+#include "photonbacktrackerservice.fcl"
+#include "mccheatermodules.fcl"
+#include "simulationservices_icarus.fcl"
+
+#include "mcsproducer.fcl"
+#include "rangeproducer.fcl"
+#include "calorimetry_icarus.fcl"
+#include "scecorrections.fcl"
+#include "trackfindermodules_icarus.fcl"
+
+process_name: CaloAna
+
+services:
+{
+  # @table::icarus_detsim_services
+  RandomNumberGenerator: {} #ART native random number generator
+  TFileService: { 
+    fileName: "calorimetryout.root"
+  }
+  message:      { debugModules: ["*"] destinations: { debugmsg:{type: "cout" threshold: "INFO"} } }
+  TimeTracker:  { printSummary: true }
+  AuxDetExptGeoHelperInterface: { service_provider: "icaruscode/CRT/CRTGeometryHelper" }
+  BackTrackerService:  @local::standard_backtrackerservice
+  ParticleInventoryService: @local::standard_particleinventoryservice
+  @table::icarus_basic_services
+  @table::icarus_backtracking_services
+}
+services.SpaceCharge: @local::icarus_spacecharge
+services.SpaceCharge.EnableCalEfieldSCE: true
+services.SpaceCharge.EnableCalSpatialSCE: true
+services.SpaceCharge.EnableSimEfield: true
+services.SpaceCharge.EnableSimEfieldSCE: true
+services.SpaceCharge.EnableSimSpatialSCE: true
+services.SpaceCharge.EnableCorrSCE: true
+services.SpaceCharge.InputFilename: "SCEoffsets/SCEoffsets_ICARUS_E500_voxelTH3.root"
+services.SpaceCharge.RepresentationType: "Voxelized_TH3"
+services.SpaceCharge.service_provider: SpaceChargeServiceICARUS
```

Comment:
How is `SpaceCharge` different from `SpaceChargeService`?

As an aside, I have opened an issue with `icaruscode`, SBNSoftware/icaruscode#236, that would be relevant here since there is way too much boilerplate configuration. @grayputnam's help in defining the content re that issue would be welcome.